source/Clipboard.js

```diff
@@ -287,7 +290,17 @@ var onDrop = function ( event ) {
         }
     }
 
+    // Try our best to get the location of the insertion
+    if ( ownerDocument && ownerDocument.caretRangeFromPoint ) {
```

Comment:
Does this work in IE? Looks like neither caretRangeFromPoint nor caretPositionFromPoint are supported.